src/GraphQLSchemaLinter.php

```diff
@@ -79,10 +79,12 @@ public function getVersion() {
     if (!empty($this->dependencyVersions)) {
 
       foreach ($this->dependencyVersions as $name => $required) {
-        $binary_path = $this->getDefaultBinary();
-        $pos = strrpos($binary_path, '/');
-        $binary_path_parent = $pos === false ? $url : substr($binary_path, 0, $pos + 1);
-        list($err, $dep_version, $stderr) = exec_manual('cd %s 2>&1 >/dev/null && npm list --depth=0 2>/dev/null | grep "%C" | cut -d "@" -f 3 | tr -d "[:space:]"', $binary_path_parent, $name);
+        $bin_root = dirname($this->getDefaultBinary());
+        $future = new ExecFuture('npm list -s --depth=0 --json 2>&1 2>/dev/null || exit 0');
```

Comment:
Do you still need to redirect to `/dev/null` now that you're just capturing `$stdout`?

Reply:
You are right, I don't. But I will still need `exit 0` because this command finishes with a non-zero code for some reason.